### DISCUSSION TITLE
Logging facebook account.

### DISCUSSION THREAD

**Discussion by Abbatii** [thatskymod/Sky-CotL-Scripts#157] [Q&A]
Please does anyone have a solution to this? My account is linked with Facebook and canvas won't allow me to log with Facebook at all.
Thank you

**Comment by thatskymod** [comment]
I’m not trying to be difficult, but let me suggest using a different login method. Google, Nintendo, and Local have proven to be reliable.

It's unclear why the Facebook option is still offered when it doesn’t function properly. I understand there may be challenges integrating Sky’s OAuth with a third-party package like Canvas, but that doesn’t justify listing it as an available option. You could raise this with the core [development team](https://github.com/artdeell/Canvas-Open-Source).

In the meantime, switching to another login provider should resolve the issue.
